ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -838,6 +883,7 @@ static PrjFS_Result HandleHydrateFileRequest(const MessageHeader* request, const
 static PrjFS_Result HandleFileNotification(
     const MessageHeader* request,
     const char* path,
+    const char* fullPath_opt,
```

Comment:
What is `_opt`? Optional?

Reply:
And does it need to be optional?

Reply:
> What is _opt? Optional?

Yep, and you're right we can just make this required.